modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -39,6 +39,43 @@ This column is returned by default.
 
 The column returns the name of the composite database that the alias belongs to, or `null` if the alias does not belong to a composite database.
 
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+IS [NOT] :: <TYPE>
+----
+a|
+
+Extended xref:syntax/expressions.adoc#type-predicate-expressions[type predicate expressions].
+Closed dynamic union types (`type1 \| type2 \| ...`) are now supported, allowing for queries such as:
+
+[source, cypher, role="noheader"]
+----
+IS :: INTEGER \| FLOAT
+----
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+CREATE CONSTRAINT name FOR (n:Label) REQUIRE n.prop IS :: <PROPERTY TYPE>
+
+CREATE CONSTRAINT name FOR ()-[r:TYPE]-() REQUIRE r.prop IS :: <PROPERTY TYPE>
+----
+a|
+
+Extended xref::constraints/examples.adoc#constraints-examples-node-property-type[node] and xref::constraints/examples.adoc#constraints-examples-relationship-property-type[relationship] property type constraints.
```

Comment:
No update to these pages?

Reply:
I was in two minds about adding some union examples. Have added a couple now.